### DISCUSSION TITLE
about canva pro

### DISCUSSION THREAD

**Discussion by arshanashok70** [community/community#30711] [GitHub Education]
is canva pro available in github student pack....

**Comment by LumitoLuma** [comment]
Yes, it is. For one year.

**Reply by arshanashok70** [reply]
But there is no option shows to activate why??
 Can you please tell how to activate.

**Reply by LumitoLuma** [reply]
Oh, seems to got removed. I found this guide: https://hackmd.io/@ll-studio/rJGNDHHxq. Haven't tried it, but may help you.

**Comment by arshanashok70** [comment]
It's not working for me ....🥺
Can you prefer any other apps or softwares like canva available in github
students pack??

**Reply by LumitoLuma** [reply]
Do you mean if there are any Canva-related apps in GitHub students pack? You may take a look [here](https://education.github.com/pack/offers/), but I didn't find any similar options. You can always use Canva free, which is still great.
